src/OAuth/OAuth2/Service/Facebook.php

```diff
@@ -197,6 +197,7 @@ public function getDialogUri($dialogPath, array $parameters)
         $query = http_build_query($parameters);
         return new Uri($baseUrl . '?' . $query);
     }
+
```

Comment:
I wasn't going to be so petty before to ask you to add an extra line... But yes that did bug me so thanks for picking up on that.